CIPs/CIP-69/CIP-69.md

```diff
@@ -44,23 +44,23 @@ The merkle tree should be constructed as a balanced tree. See the [implementatio
 
 
 ### Leaf sorting
-Before creating the tree the leafs (updates) should be sorted. The sorting should be based on a few different properties of the Ceramic document that is being updated and the update record itself:
+Before creating the tree the leafs (updates) should be sorted. The sorting should be based on a few different properties of the Ceramic document that is being updated:
 
-1. `topic` - sort by the topic in the document metadata
+1. `collection` - sort by the topic in the document metadata
 2. `schema` - if multiple documents have the same topic, sort by the *schema*
-3. `DID` - if multiple documents have the same topic and schema, extract and sorty by the DID from the `kid` in the signed record
+3. `controllers` - if multiple documents have the same topic and schema, sort by the first controller, then subsequent ones
```

Comment:
For this to work right, we'll have to enforce that the `controllers` array in records is always stored sorted, regardless of the order the user specifies them in.  Should we make a story for this now so we don't forget?  Is there an epic for supporting multiple controllers we could put it in?

Reply:
I guess that's true, unless you already know the order from a previous version of the document. I don't think there is an epic yet, but feel free to create one1

Reply:
https://github.com/ceramicnetwork/js-ceramic/issues/514